### ISSUE TITLE
Sample module gives strange AAI values

### ISSUE THREAD

**Issue by SWittouck** [SWittouck/SCARAP#19]
On a dataset with ~ 4,000 Lactobacillaceae genomes, some very closely related, all cAAI values (using ~ 400 core genes) were below 94%. This is strange; at least some of these values should be very high. 

Also: the README currently specifies "ANI" calculation in the sample module instructions, but uses the faa files. This is confusing.

**Comment by TomEile** [comment]
The guideline for subsampling a set of genomes should not include the ./faas but should include the ./ffns, so that you use the cANI instead of cAAI

**Comment by SWittouck** [comment]
I did some research and it seems that the low AAI values are not problematic. They are caused by the use of "alignment mode 1" in the MMseqs align module, which produces quick estimates of sequence identity values from per-column alignment scores. This is faster than computing exact alignments, but yields on average lower values for AAIs above ~ 90%. See [the MMseqs2 wiki page](https://github.com/soedinglab/MMseqs2/wiki#how-does-mmseqs2-compute-the-sequence-identity). 

I added a disclaimer on this topic to the SCARAP README file.
